lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -367,6 +367,19 @@ def self.enforce_fallback(allowed_values: nil, fallback: nil)
           :allowed_from_server => false,
           :description => 'If `false`, all LLM (OpenAI) instrumentation will be disabled and no metrics, events, or spans will be sent. AI Monitoring is automatically disabled if `high_security` mode is enabled.'
         },
+        :'ai_monitoring.record_content.enabled' => {
+          :default => true,
+          :public => true,
+          :type => Boolean,
+          :allowed_from_server => false,
+          :description => <<~DESCRIPTION
+            If `false`, LLM (OpenAI) instrumentation will not capture input and output content on specific LLM events.
```

Comment:
The description states that this option is specific to LLM and even more specific to OpenAI at that. My first instinct would be to express that specificity in the config parameter naming. Is there a strategy behind not having "LLM" nor "OpenAI" in the name, such as keeping it generic for future-proofing?

Reply:
Yes for future-proofing! This will include Bedrock at some point

Reply:
In that case I'm torn between changing `"(OpenAI)"` to `"(OpenAI only for now)"` or simply referring to LLM instrumentation generically now in a future proof way that doesn't mention a vendor at all:

```suggestion
            If `false`, LLM instrumentation will not capture input and output content on specific LLM events.
```

Reply:
I liked having the OpenAI reference since I think this was the only place to specifically call out which instrumentations are impacted by this config change. We're okay with the "(OpenAI only for now)" route?

Reply:
I'm fine with "OpenAI only for now". I prefer that over removing "OpenAI" entirely.

Reply:
Definitely go with whatever you both like and makes the most sense to you (given that you're much closer to its usage than I am). My thing was a) LLM is much bigger than OpenAI, and b) AI is much bigger than LLM, and c) this option is described as referring to OpenAI LLM without mentioning either of those words in the parameter name. If you're telling me that the parameter name itself will not need to change in future, I'm fine with the description saying whatever you like for now and getting updated in future.

The reason I advocate for "LLM (OpenAI only for now)" vs "LLM (OpenAI)" is that it wasn't immediately clear to me what the bit in parens was clarifying. Perhaps "(OpenAI) LLM" would have made more sense in my brain.

Reply:
Changed to "OpenAI only for now" https://github.com/newrelic/newrelic-ruby-agent/pull/2482/files#diff-d116d3c8c7bebe204a18ee3983ac59c5d15bd70da7123f2a2c4a12f144bca181R368